backend/listener.js

```diff
@@ -167,7 +177,17 @@ const handleNewHead = (head, networkId) => {
   const number = web3.utils.hexToNumber(head.number)
   const timestamp = web3.utils.hexToNumber(head.timestamp)
 
-  Network.upsert({ networkId, lastBlock: number })
+  Network.update(
+    { networkId, lastBlock: number },
```

Comment:
you did not change it but I wonder why is it necessary to update the networkId?

Reply:
Sequelize will insert a row with a null `network_id` if it's not included...